jupyterlab_git/git.py

```diff
@@ -502,12 +512,14 @@ async def log(self, path, history_count=10, follow_path=None):
 
             if is_single_file:
                 commit["is_binary"] = line_array[i + 4].startswith("-\t-\t")
+                # [insertions, deletions, original_file_path?, current_file_path]
+                commit["file_path"] = line_array[i + 4].split()[-1]
```

Comment:
Here you will need to deal with the rename case (aka is there some null character in `line_array[i + 4]`)